cla-backend/cla/models/github_models.py

```diff
@@ -147,8 +147,19 @@ def oauth2_redirect(self, state, code, request): # pylint: disable=too-many-argu
         """
         cla.log.info('Handling GitHub OAuth2 redirect')
         session = self._get_request_session(request)
-        if state != session.get('github_oauth2_state', None):
-            cla.log.warning('Invalid GitHub OAuth2 state')
+
+        cla.log.info('State: %s', state)
```

Comment:
Can make this `cla.log.debug` if you want, not required - but should work since we have DEBUG logging turned on for now.